c2corg_api/models/feed.py

```diff
@@ -448,6 +463,24 @@ def update_activities_of_changes(document):
     )
 
 
+def update_languages_of_changes(document):
+    """Update the langs of all feed entries of the given document.
+    """
+
+    lang_ids = DBSession. \
+        query(func.array_agg(DocumentLocale.lang,
+                             type_=postgresql.ARRAY(String))). \
+        filter(DocumentLocale.document_id == document.document_id). \
+        group_by(DocumentLocale.document_id). \
+        scalar()
```

Comment:
Perhaps it's possible to use ``subquery()`` instead to avoid having 2 distinct SQL queries?